answers.js

```diff
@@ -0,0 +1,15 @@
+Specifications
+/*1: Guess The Number
+Write a program that will generate a number between 1 and 100. Your program will ask the user to guess the generated number.When the user types a guess, the program should tell them if they won (the guess matched the computer's number), if their guess was higher than the number the computer generated, OR if the guess was lower than the number the computer generated. Terminate the program if the user types "exit".*/
+function(randomno) {
+  var compuguess = computer guess
```

Comment:
What is computer guess? Are you supposed to get any values from this variable or is it a string? If it's a string it should be wrapped in quotes.